raw-cms-app/src/modules/core/views/collection-table-view/collection-table-view.js

```diff
@@ -15,7 +16,9 @@ const _CollectionTableView = async (res, rej) => {
         return this.$route.params.collName;
       },
       title: function() {
```

Comment:
The `title` computed property is not needed anymore (is not used in the template): you can add the `mounted` lifecycle hook for Vue components and just update the top bar title via
```javascript
vuexStore.dispatch('core/updateTopBarTitle', this.$t('core.collections.table.title', { name: this.collectionName }));
```